src/main/antlr4/org/reaktivity/nukleus/maven/plugin/internal/parser/Nukleus.g4

```diff
@@ -191,15 +201,23 @@ string16_type
    ;
 
 int_literal
-   : INTEGER_LITERAL
+   : negative_int_literal
+   | uint_literal
+   ;
+
+negative_int_literal
+   : '-' UNSIGNED_INTEGER_LITERAL
+   ;
+
+uint_literal
+   : UNSIGNED_INTEGER_LITERAL
    | HEX_LITERAL
    ;
 
```

Comment:
```
int_literal
  : MINUS ? uint_literal
```
...
```
MINUS
  : '-'
  ;
```

Reply:
I don't think we should allow minus sign with hex literal. Are you saying we should (or could)?

Reply:
done, we'll accept - with hex literal since Integer.decode does